webapp/src/components/rhs/rhs_run_details.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import Scrollbars from 'react-custom-scrollbars';
+import {useSelector} from 'react-redux';
+
+import {GlobalState} from 'mattermost-redux/types/store';
+
+import {PlaybookRun} from 'src/types/playbook_run';
+import {
+    renderThumbHorizontal,
+    renderThumbVertical, renderView,
+    RHSContainer,
+    RHSContent,
+} from 'src/components/rhs/rhs_shared';
+import {currentPlaybookRun} from 'src/selectors';
+import RHSAbout from 'src/components/rhs/rhs_about';
+import RHSChecklists from 'src/components/rhs/rhs_checklists';
+
+const RHSRunDetails = () => {
+    const playbookRun = useSelector<GlobalState, PlaybookRun | undefined>(currentPlaybookRun);
```

Comment:
We can remove the type here, but I don't know if you actually want it there for being super explicit? I think I prefer readability over specificity here, but 1/5.

```suggestion
    const playbookRun = useSelector(currentPlaybookRun);
```

Reply:
Awesome, done. Agree, I prefer implicit types for that.